osu.Game/Overlays/Profile/Header/Components/DailyChallengeStatsTooltip.cs

```diff
@@ -138,34 +138,32 @@ public void SetContent(DailyChallengeTooltipData content)
             topFifty.ValueColour = colourProvider.Content2;
         }
 
-        // reference: https://github.com/ppy/osu-web/blob/adf1e94754ba9625b85eba795f4a310caf169eec/resources/js/profile-page/daily-challenge.tsx#L13-L47
+        // reference: https://github.com/ppy/osu-web/blob/a97f156014e00ea1aa315140da60542e798a9f06/resources/js/profile-page/daily-challenge.tsx#L13-L47
 
-        // Rounding up is needed here to ensure the overlay shows the same colour as osu-web for the play count.
-        // This is because, for example, 31 / 3 > 10 in JavaScript because floats are used, while here it would
-        // get truncated to 10 with an integer division and show a lower tier.
-        public static RankingTier TierForPlayCount(int playCount) => TierForDaily((int)Math.Ceiling(playCount / 3.0d));
+        // Rounding down is needed here to ensure the overlay shows the same colour as osu-web for the play count.
```

Comment:
This follows suit with osu-web code 1:1 so this comment does not have to exist anymore. Previously it was there because osu-web doesn't look to be doing any ceiling, while it does internally.